app.py

```diff
@@ -1,13 +1,19 @@
 from flask_api import FlaskAPI
+from flask import request
 
 app = FlaskAPI(__name__)
 
 @app.route('/data', methods=['PATCH'])
 def data():
     req = request.get_json()
+    
+    key = list(req)[0]
+    old_value = RESULT[key]
```

Comment:
nếu key này ko tồn tại, đoạn code này sẽ dẫn đến thêm một key+value mới vào dict -> undesired state mutation.
Fix it: if key doesnt exist, report back to client.